src/jazkarta/easyformplugin/savecontent/browser.py

```diff
@@ -16,3 +21,34 @@ def has_saved_content(self):
 
     def saved_content_url(self):
         return self.context.absolute_url() + '/' + STORAGE_ID
+
+
+class CSVDownload(BrowserView):
+
+    def __call__(self):
+        items_dicts = []
+        keys = OrderedDict()
+        for obj in self.context.objectValues():
+            behavior_assignment = EasyformSchemaBehaviorAssignment(obj)
+            obj_dict = {}
+            for field_name in behavior_assignment.schema():
+                obj_dict[field_name] = getattr(obj, field_name)
+                keys[field_name] = True
+            items_dicts.append(obj_dict)
+        output = StringIO()
+        csv_writer = csv.DictWriter(output, keys.keys())
+        csv_writer.writeheader()
+        for item in items_dicts:
+            csv_writer.writerow(item)
+        self.request.response.setHeader("Content-type", "text/csv")
+        self.request.response.setHeader("Content-Disposition", 'attachment; filename="%s.csv"' % behavior_assignment.find_form().title)
+        return output.getvalue()
+
+
+class ButtonViewlet(ViewletBase):
```

Comment:
A button in a viewlet is probably fine, but I wonder if an action (either in `object_buttons` or `document_actions`) on the content type FTI would be better here?